### ISSUE TITLE
Task: Remove Yoast dashboard widget

### ISSUE THREAD

**Issue by jazzsequence** [humanmade/altis-seo#52]
By default, Yoast SEO adds a dashboard widget (which includes custom CSS and JS). We don't need this in the Altis Dashboard so we can remove.

**Acceptance Criteria:**
- [ ] Remove Dashboard widget
  - [ ] dequeue dashboard scripts
  - [ ] dequeue dashboard styles

**Dev Notes**
example code:
```php
	add_action( 'admin_init', function() {
		// Remove the Dashboard widget.
		remove_meta_box( 'wpseo-dashboard-overview', 'dashboard', 'normal' );
		wp_dequeue_script( 'dashboard-widget' );
		wp_dequeue_style( 'wp-dashboard' );
	}
```

**Comment by roborourke** [comment]
The names of that script and style winds me up :P

**Comment by jazzsequence** [comment]
`¯\_(ツ)_/¯` 😬

**Comment by roborourke** [comment]
Fixed by #65
